docs/APP_DEVELOPMENT.md

```diff
@@ -1,15 +1,14 @@
-#  Developing OVE applications
+# Developing OVE applications
 
-OVE provides a number of apps to display commonly-used types of content, such as HTML, tiled images, maps, networks, and charts.
-The HTML app is particularly flexible, and allows the hosting of general HTML/JavaScript web applications.
-However, if existing apps do not meet your needs then you can write a new OVE app.
+OVE provides a number of applications to display commonly-used types of content, such as HTML, tiled images, audio and video files, maps, networks, and charts.
+The HTML application is particularly flexible, and allows the hosting of general HTML/JavaScript web applications.
+However, if existing applications do not meet your needs then you can write a new OVE app.
 
-[``ove-lib-appbase``](https://github.com/ove/ove/tree/master/packages/ove-lib-appbase) provides a base library on whuch OVE apps can be built.
+[`@ove-lib/appbase`](https://www.npmjs.com/package/@ove-lib/appbase) provides a base library on which OVE applications can be built.
```

Comment:
"[``ove-lib-appbase``](https://www.npmjs.com/package/@ove-lib/appbase) **([code on GitHub](https://github.com/ove/ove/tree/master/packages/ove-lib-appbase))** provides a"

i.e., add

    ([code on GitHub](https://github.com/ove/ove/tree/master/packages/ove-lib-appbase)

Reply:
NPM provides a link to GitHub and there is also no possibility for someone to use the module without adding it via NPM. So, the link to the code IMHO is redundant and I did not add this suggestion in my fix.

Reply:
Yes, but this requires several extra navigation steps:
1) go from NPM page to GitHub which takes you to `https://github.com/ove/ove`
2) navigate to the `packages/` subdirectory
3) navigate to the ` 	ove-lib-appbase/` subdirectory

This is much less convenient for someone who wants to quickly see what the package provides.

Reply:
Will fixing the landing page on NPM to have a direct link to the source (i.e. sub-directory) fix that?